gridscout

```diff
@@ -110,7 +142,7 @@ class M:
                 if 'results' in response:
                     results = response['results']
                     self.print_results(results)
-                    print ''
+                    print '\n'
```

Comment:
Since some things are too wide to fit on a screen, they leak over into the next line. This extra line feed makes it obvious that it is an intentional gap between sets.